reactor-client/src/main/java/se/fortnox/reactivewizard/reactorclient/ReactorRequestBuilder.java

```diff
@@ -0,0 +1,26 @@
+package se.fortnox.reactivewizard.reactorclient;
+
+import io.netty.handler.codec.http.HttpMethod;
+import se.fortnox.reactivewizard.client.RequestBuilder;
+
+import java.net.InetSocketAddress;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public class ReactorRequestBuilder extends RequestBuilder {
+    private String content = "";
+
+    public ReactorRequestBuilder(InetSocketAddress serverInfo, HttpMethod method, String key) {
+        super(serverInfo, method, key);
+    }
+
+    public String getContent() {
```

Comment:
Wasn't the base implementation using byte[] instead of String? Converting to String is very poor in java, leads to byte inflation, can you use byte[]?

Reply:
> Wasn't the base implementation using byte[] instead of String? Converting to String is very poor in java, leads to byte inflation, can you use byte[]?

Yes, will fix this